src/utils/stats.ts

```diff
@@ -0,0 +1,154 @@
+interface Exercise {
+	[exerciseName: string]: Section;
+}
+
+interface Stats {
+	[gender: string]: Exercise[];
```

Comment:
I'd change this to sex just to make it consistent across all files.

Reply:
Changing it here, but if we ae using it across multiple files, I'll suggest creating a separate file for types